src/triage/component/catwalk/storage.py

```diff
@@ -40,6 +41,8 @@ def write(self, obj):
         s3 = s3fs.S3FileSystem()
         with s3.open(self.path, 'wb') as f:
             joblib.dump(obj, f, compress=True)
+        # Return the file size of the model object in kB
+        return s3.info(self.path)["Size"]/(1024.0)
```

Comment:
I would actually prefer size to be a readable property instead of only available on write. It can throw a RuntimeError if called while self.exists is false.